src/Nav/index.js

```diff
@@ -30,25 +27,15 @@ const Navbar = ({ enableSearch, siteID, siteTitle }) => {
     }
   };
 
+
   return (
     <BreakpointProvider>
-      <Context.Provider value={{ siteTitle, siteID }}>
-        <nav aria-label="Libra cross-domain nav" className={styles.root}>
-          <PrimaryNav activePopupMenu={activePopupMenu} setPopupMenu={setPopupMenu} />
-            <SubNav 
-              activePopupMenu={activePopupMenu} 
-              enableSearch={enableSearch}
-              setPopupMenu={setPopupMenu} 
-            />
-        </nav>
-      </Context.Provider>
+      <nav aria-label="Libra cross-domain nav" className={styles.root}>
+        <PrimaryNav activePopupMenu={activePopupMenu} setPopupMenu={setPopupMenu} />
```

Comment:
I know this was not really changed in this PR but move the `activePopupMenu` and `setPopupMenu` to a const. and pass that in to each component.

```
const navProps = {
  prop1: 'something',
  prop2: 'something else'
};

<PrimaryNav {...navProps} />
```